package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "create-and-manage-post",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
```

Comment:
Popper.js seems to be a dependency of bootstrap.

<img width="851" alt="Screen Shot 2023-11-01 at 8 23 44 PM" src="https://github.com/NdongmoJerry/create-and-manage-post/assets/16974765/d355b795-1433-414d-ac40-2bfb80a6f0de">

I sorted it out with
`npm install popper.js --save`